siva/location.go

```diff
@@ -24,9 +26,20 @@ type Location struct {
 
 var _ borges.Location = (*Location)(nil)
 
-// NewLocation creates a new Location object.
-func NewLocation(id borges.LocationID, lib *Library, path string) (*Location, error) {
-	cp, err := newCheckpoint(lib.fs, path)
+// OpenLocation opens a location and returns a Location struct.
+func OpenLocation(id borges.LocationID, lib *Library, path string) (*Location, error) {
```

Comment:
This new public method is used nowhere, should we keep it anyway?

Reply:
We probably get rid of both `CreateLocation` and `OpenLocation` and use only `NewLocation` with the create parameter.

Reply:
Even we could make `NewLocation` private because as library user you should use the `Library` to manage locations.